### ISSUE TITLE
Get rid of column limit in markdown files

### ISSUE THREAD

**Issue by ISSOtm** [gbdev/pandocs#333]
The column limit made sense in the original document, since it was read in its raw source form. However, modern Pan Docs are instead processed.

The column limit is currently not respected everywhere, anyway... and it's actually harmful, since it means that changes are not localized to where changes are actually made, but also bleed into neighboring lines when re-balancing everything.

Since (single) line breaks are entirely ignored in Markdown (and mdBook certainly does), I suggest we split semantically instead, i.e. one line break after every sentence. 80-column editors are rare nowadays, and soft-wrapping what is mostly prose isn't nearly as jarring as doing so with code.

**Comment by aaaaaa123456789** [comment]
Dropping the 80-column limit makes sense. There's no good reason to have such a limit anywhere nowadays.
That being said, I'm not sure if no limit is a good idea. I try to limit myself to 120ish for text documents; it keeps them editable and readable in source form.

**Comment by ISSOtm** [comment]
The limit would be "soft"-enforced by the requirement to keep sentences reasonably short. Though, again, I have soft-wrapping enabled for all text documents, and it works very well.

**Comment by avivace** [comment]
I disagree, a limit is still useful to keep the source files readable and editable. We can discuss about increasing the limit to 100-120
